src/atoms/Link/Link.web.js

```diff
@@ -1,3 +1,70 @@
-const Link = () => null;
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import { getColor } from '../../_helpers/theme';
+import automation from '../../_helpers/automation-attributes';
+
+const StyledText = styled(Text)`
+  color: ${(props) => getColor(props.theme, 'primary.800')};
+  text-decoration: none;
+  cursor: pointer;
+  &:hover {
+    text-decoration: underline;
+    color: ${(props) => getColor(props.theme, 'primary.800')};
+  }
+  &:focus {
+    outline: none;
+    text-decoration: underline;
+    color: ${(props) => getColor(props.theme, 'primary.900')};
+  }
+  &:active {
+    outline: none;
+    text-decoration: underline;
+    color: ${(props) => getColor(props.theme, 'primary.700')};
+  }
+  &:visited {
+    outline: none;
+    color: ${(props) => getColor(props.theme, 'primary.700')};
+  }
+`;
+
+const Link = ({ size, children, href, target, testID, rel }) => {
+  return (
+    <StyledText as="a" rel={rel} size={size} href={href} target={target} {...automation(testID)}>
+      {children}
+    </StyledText>
+  );
+};
+
+Link.propTypes = {
```

Comment:
We'll be adding `disabled` prop as well.

Reply:
will update it once done

Reply:
Please re-request for review once `disabled` prop is implemented.

Reply:
Disabled not implementing now. Let the designers come back with the colors and everything

Reply:
Alright. Created an issue #101 so we don't forget.

Reply:
@susheelachoudry Designs have been added for Link disabled. Please check and update the PR.